src/features/footer/data/footerLayoutSlice.ts

```diff
@@ -0,0 +1,38 @@
+import { fetchFooterLayoutSaga } from 'src/features/footer/data/fetchFooterLayoutSagas';
+import { createSagaSlice } from 'src/shared/resources/utils/sagaSlice';
+import type {
+  IFetchFooterLayoutFulfilled,
+  IFooterLayoutActionRejected,
+  IFooterLayoutState,
+} from 'src/features/footer/data/types';
+import type { MkActionType } from 'src/shared/resources/utils/sagaSlice';
+
+export const initialState: IFooterLayoutState = {
+  footerLayout: null,
+  error: null,
```

Comment:
Is this ever read? What happens if the footer API responds with a catastrophic failure? Hint: Look into `getHasErrorsSelector`

Reply:
Aha, maybe did it a little to quickly 😅